src/style.css

```diff
@@ -1 +1,83 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  width: 100%;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  width: 1200px;
+}
+
+.logo {
+  display: flex;
+  height: 40px;
+  width: 40px;
+  cursor: pointer;
+  align-self: center;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+  text-transform: uppercase;
+}
+
+.nav__item {
+  height: 60px;
+  display: flex;
+}
+
+.nav__item:nth-child(-n+7) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  font-family: "Roboto", sans-serif;
+  position: relative;
+  align-content: center;
+  align-items: center;
+  display: flex;
+  color: #000;
+  font-weight: 500;
+  font-size: 12px;
+  text-decoration: none;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__link:active::after {
```

Comment:
![image](https://user-images.githubusercontent.com/70009205/211631944-dc46511d-39ad-4e0b-a5cc-af3ec7110e6a.png)
not fixed